version 1.1

EvgeniyKuch opened the pull request:
Приложение, хранящее информацию о книгах в библиотеке

Использованы возможности Spring Data для подключения к нереляционным базам данных.<br>
Web-интерфейс выполнен на Thymeleaf.<br>
Получение списка книг реализовано с помощью jQuery и REST-контроллера.<br>

Хранение данных выполняется в Mongo DB.
Структуру и начальные данные создаёт Mongock.<br>

Commit:
version 1.1

Приложение, хранящее информацию о книгах в библиотеке

Использованы возможности Spring Data для подключения к нереляционным базам данных.<br>
Web-интерфейс выполнен на Thymeleaf.<br>
Получение списка книг реализовано с помощью jQuery и REST-контроллера.<br>

Хранение данных выполняется в Mongo DB.
Структуру и начальные данные создаёт Mongock.<br>

Commit:
version 1.2

Исправлен контроллер полчения списка книг. Убра ненужный функуционал.

kataus commented on library-spring-data-mongo-mvc-view-rest/src/main/java/ru/otus/library/controller/ExceptionController.java:
```diff
@@ -0,0 +1,16 @@
+import ru.otus.library.controller.exceptions.NotFoundException;
+
+@ControllerAdvice
+public class ExceptionController {
```
это всё таки не контроллер

kataus commented on library-spring-data-mongo-mvc-view-rest/src/main/java/ru/otus/library/controller/HomeController.java:
```diff
@@ -0,0 +1,14 @@
+public class HomeController {
+
+    @GetMapping("/")
+    String getHome(Model model) {
```
А почему выбран default как область видимости метода?

Commit:
version 1.3

Изменена область видимости метода getHome. Обработчик исключений перемещен в  другой пакет.
